discord/types/guild.py

```diff
@@ -177,11 +176,8 @@ class ChannelPositionUpdate(TypedDict):
     parent_id: Optional[Snowflake]
 
 
-class _RolePositionRequired(TypedDict):
-    id: Snowflake
-
-
-class RolePositionUpdate(_RolePositionRequired, total=False):
+class RolePositionUpdate(TypedDict, total=False):
+    id: Required[Snowflake]
```

Comment:
Unsure as to why this was formatted opposite of all the others, but I kept the existing schema anyways.